wtxmgr/unconfirmed.go

```diff
@@ -83,28 +94,34 @@ func (s *Store) removeConflict(ns walletdb.ReadWriteBucket, rec *TxRecord) error
 	// For each potential credit for this record, each spender (if any) must
 	// be recursively removed as well.  Once the spenders are removed, the
 	// credit is deleted.
-	numOuts := uint32(len(rec.MsgTx.TxOut))
-	for i := uint32(0); i < numOuts; i++ {
-		k := canonicalOutPoint(&rec.Hash, i)
-		spenderHash := existsRawUnminedInput(ns, k)
-		if spenderHash != nil {
+	for i := range rec.MsgTx.TxOut {
+		k := canonicalOutPoint(&rec.Hash, uint32(i))
+		spenderHashes := fetchUnminedInputSpendTxHashes(ns, k)
+		for _, spenderHash := range spenderHashes {
+			spenderVal := existsRawUnmined(ns, spenderHash[:])
+			// It might be the case that this spending transaction
```

Comment:
nit: add newline

Reply:
Similar comment here about updating the comment to reflect reality.

Reply:
Fixed.